docs/awesome.md

```diff
@@ -114,7 +114,7 @@ description: A curated list of useful tools, libraries, gamemodes, filterscripts
 ## Crypto Plugins
 
 - **[whirlpool](https://github.com/Southclaws/samp-whirlpool)** - Whirlpool Cryptographic plugin for SA:MP.
-- **[bcrypt](https://github.com/LassiR/bcrypt-samp)** - Bcrypt Cryptographic plugin for SA:MP.
+- **[bcrypt](https://github.com/Sreyas-Sreelal/samp-bcrypt)** - Bcrypt Cryptographic plugin for SA:MP.
```

Comment:
Why you replaced the link of the lassir's bcrypt plugin to Sreyas-Sreelal's?

Reply:
This one is maintained

Reply:
I agree with this, if @adib-yg also agrees we will merge.